packages/core/upload/server/src/controllers/validation/admin/folder-file.ts

```diff
@@ -29,6 +29,7 @@ const validateStructureMoveManyFoldersFilesSchema = yup
 
 const validateDuplicatesMoveManyFoldersFilesSchema = yup
   .object()
+  // eslint-disable-next-line func-names
   .test('are-folders-unique', 'some folders already exist', async function (value) {
```

Comment:
With an arrow function this wouldn't be required

Reply:
I preferred to solve the problem by giving a name to the function